docs/get-started/using-standalone.md

```diff
@@ -42,15 +42,21 @@ const deckgl = new Deck({
 
 ## Using the Scripting API
 
-deck.gl also offers a standalone bundled version of the library - a native JavaScript scripting interface like that of d3.js. You can now use deck.gl in prototype environments such as [Codepen](https://codepen.io), [JSFiddle](https://jsfiddle.net) and [Observable](https://observablehq.com). This effort aims to make it easier for designers, creative coders and data scientists everywhere to leverage the GPU for interactive visualizations. The scripting API offers out-of-the-box integration with [Mapbox GL JS](https://mapbox.com). To add a base map to your visualization, you need to include the base map library and stylesheet:
+deck.gl also offers a standalone bundled version of the library - a native JavaScript scripting interface like that of d3.js. You can now use deck.gl in prototype environments such as [Codepen](https://codepen.io), [JSFiddle](https://jsfiddle.net) and [Observable](https://observablehq.com). This effort aims to make it easier for designers, creative coders and data scientists everywhere to leverage the GPU for interactive visualizations. The scripting API offers out-of-the-box integration with [Mapbox GL JS](https://mapbox.com) or [MapLibre GL JS](https://maplibre.org). To add a base map to your visualization, you need to include the base map library and stylesheet:
 
 To use deck.gl in a scripting environment, include the standalone version in a `script` tag:
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="map-library">
+  <TabItem value="mapbox" label="Mapbox">
```

Comment:
We use tabs in `react-map-gl`, I think it makes sense to use here as well.
<img width="1060" alt="Screen Shot 2024-03-06 at 3 15 02 PM" src="https://github.com/visgl/deck.gl/assets/2461547/548e45a5-a52f-4f71-a81f-c1ff6ef38e82">